code/game/machinery/cryopod.dm

```diff
@@ -276,7 +276,7 @@
 		qdel(I)
 	qdel(R.module)
 
-	return ..()
+	. = ..()
```

Comment:
Have you reproduced this bug, and tested if it fixed? Because to me this looks like you only copied solution from baystation's repo, which actually does the same in this scenario. Because if not, this PR not only close the issue of the bug which isn't fixed, but also does nothing.
But in the end this is byond.. so, it will be magic, if this resolved issue.